src/producer/partitioners/defaultJava/Test.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Arrays;
+import java.util.List;
```

Comment:
Can you ignore this file on the `.npmignore`, otherwise it will be in the bundle

Reply:
I thought I did. I accidentally fucked up git a bit and had to do some reflog madness, so it got lost 🙈

Reply:
Fixed in 12c92c0